logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/auth/sso/SSOLoginScope.kt

```diff
@@ -0,0 +1,15 @@
+package com.wire.kalium.logic.feature.auth.sso
+
+import com.wire.kalium.logic.data.auth.login.SSOLoginRepository
+import com.wire.kalium.logic.feature.auth.ValidateUUIDUseCase
+import com.wire.kalium.logic.feature.auth.ValidateUUIDUseCaseImpl
+
+class SSOLoginScope(
+    private val ssoLoginRepository: SSOLoginRepository
+) {
+    private val validateUUIDUseCase: ValidateUUIDUseCase get() = ValidateUUIDUseCaseImpl()
+    val initiate: SSOInitiateLoginUseCase get() = SSOInitiateLoginUseCaseImpl(ssoLoginRepository, validateUUIDUseCase)
+    val finalize: SSOFinalizeLoginUseCase get() = SSOFinalizeLoginUseCaseImpl(ssoLoginRepository)
```

Comment:
For this one also I suggest `SSOLoginFinalizerUseCase`, what do you think?

Reply:
Same here as the above 😄